src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -426,7 +426,6 @@ def _create_pgm_input_lines(self):
         pgm_lines["x1"] = self._get_pp_attr("line", "x_ohm_per_km", expected_type="f8") * multiplier
         pgm_lines["c1"] = c_nf_per_km * length_km * parallel * 1e-9
         # The formula for tan1 = R_1 / Xc_1 = (g * 1e-6) / (2 * pi * f * c * 1e-9) = g / (2 * pi * f * c * 1e-3)
-        pgm_lines["tan1"] = 0.0
```

Comment:
i think your initial suggestion was actually correct, but I do agree with SonarQube that it would cause unnecessary assignments. The difficulty here lies in the fact that only valid values are broadcasted, which SonarQube is not able to understand, apparently.

Instead, maybe you can do something along the lines of

```python
invalid = np.isclose(c_nf_per_km, 0.0)
pgm_lines["tan1"] = np.divide(
    self._get_pp_attr("line", "g_us_per_km", expected_type="f8", default=0),
    c_nf_per_km * (2 * np.pi * self.system_frequency * 1e-3),
    where=np.logical_not(invalid),
)
pgm_lines["tan1"][invalid] = 0.0
```

Reply:
nvm. just read https://github.com/PowerGridModel/power-grid-model-io/pull/326#issuecomment-3279663612 again. Resolving

> @mgovers @nitbharambe there is still a reliability issue on pypi.
I've checked that 2 No. lines causing this issue can be removed, the field is auto initialised with zero even when np.divide has some False entries in where.